03-csv-vector-raster-plotting.Rmd

```diff
@@ -325,7 +329,7 @@ plot(extent(aoiBoundary_HARV),
      lwd=6,
      col="springgreen")
 
-legend("left",
+legend("bottom",
```

Comment:
Did you review these pages as built before making this change? 

http://data-lessons.github.io/NEON-R-Spatial-Vector//images/rfigs/03-csv-vector-raster-plotting/compare-extents-1.png

I placed this on the left because when you knit, the ratio scales in an odd way. You can specify the size explicitly in the chunk but i ran out of time trying to figure this out. legend left reads much better although not ideal. the best solution is figuring out xpd settings however that also knits oddly.